x-pack/plugins/security/common/licensing/license_service.test.ts

```diff
@@ -94,7 +98,7 @@ describe('license features', function() {
     }
   });
 
-  it('should show login page and other security elements, allow RBAC but forbid role mappings, DLS, and sub-feature privileges if license is basic.', () => {
+  it('should show login page and other security elements, allow RBAC but forbid role mappings, access agreement, DLS, and sub-feature privileges if license is basic.', () => {
```

Comment:
This is getting a bit out of hand 😅What do you think about something like this? 

```suggestion
  it('should show login page and other security elements, allow RBAC but forbid paid features if license is basic.', () => {
```

Reply:
:slightly_smiling_face: yeah